source/develop/release-management/features/virt-sparsify.html.md

```diff
@@ -1,53 +1,64 @@
 ---
 title: virt-sparsify
-authors: doron, shaharh
+authors: doron, shaharh, smelamud
 wiki_title: Features/virt-sparsify
-wiki_revision_count: 5
-wiki_last_updated: 2015-11-02
-feature_name: Sparse VM disks
+wiki_revision_count: 6
+wiki_last_updated: 2017-02-06
+feature_name: Sparsify VM disks
 feature_modules: engine,vdsm
-feature_status: Design
+feature_status: Released
 ---
 
 # Virt-Sparsify
 
 ## Summary
 
-This feature will enable user to run [virt-sparsify](http://libguestfs.org/virt-sparsify.1.html) on a virtual machines disks. virt-sparsify will enable to trim disk size which grows by usage.
+This feature will enable user to run [virt-sparsify](http://libguestfs.org/virt-sparsify.1.html) on a virtual machines' disks. virt-sparsify removes unused space from a disk image and returns it to the storage.
 
 ## Owner
 
-*   Name: Shahar Havivi (Shaharh)
-*   Email: <shavivi@redhat.com>
+*   Name: Shmuel Melamud
+*   Email: <smelamud@redhat.com>
 
 ## Detailed Design
 
-### vdsm
+### VDSM
 
-*   Currently vdsm support virt-sparsify in a simple form sparsing to a new disk which take more space and needs a temporary path.
-*   We want to add the usage of in place sparsing ie no need for a temporary disk.
-*   use --machine-readable for parsing the virt-sparsify output
-*   parse output and report progress
-*   ability to cancel task
+*   Currently VDSM supports virt-sparsify in a simple form that creates a temporary disk. This approach needs a temporary location and extra disk space (possibly a lot).
+*   We want to use in-place sparsification that doesn't create a temporary disk.
+*   The operation is asynchronous and implemented using storage jobs.
+*   New SDM verb is created, so the operation may be performed on any host.
```

Comment:
I agree. 
@smelamud Once you decide on content I can edit it.
You might want to start the sentence with:
"With the new approach, a temporary disk, location, and extra disk space are not needed, thanks to.."



Replacing the need for a temporary disk, the new